deps/node/test/skip_tests.txt

```diff
@@ -193,7 +194,9 @@ test/parallel/test-worker-nexttick-terminate.js
 test/parallel/test-worker-process-cwd.js
 test/parallel/test-worker-process-env.js
 test/parallel/test-worker-sharedarraybuffer-from-worker-thread.js
+test/parallel/test-worker-stack-overflow.js
 test/parallel/test-worker-stdio.js # CI 테스트중 간헐적으로 crash. 타이밍 이슈로 추정됨
+test/parallel/test-worker-stdio-from-preload-module.js # occasionally crashed in CI, might be timing issue
```

Comment:
Several tests failed occasionally in CI. 
But when I run these tests on my local environment, it passed.
I think that this is related with timing issue as these tests all employ `worker` method.